Tasks/PackerBuildV1/Strings/resources.resjson/de-de/resources.resjson

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
you don't need to check-in rejson for any other language except en-us. Others are auto-generated in a separate process.

Reply:
removed files not needed from PR